l2tdevtools/review_helpers/github.py

```diff
@@ -157,6 +158,9 @@ def GetUsername(self, access_token):
     if not response_data:
       return None
 
+    if isinstance(response_data, bytes):
```

Comment:
Please use py2to3, don't directly compare to bytes

Reply:
well, I was just using the same piece of code that was in another function in the same file...

changed